### ISSUE TITLE
Home Page—Main CTAs Need More Content

### ISSUE THREAD

**Issue by andandandandrew** [andandandandrew/HB-Site#8]
Adam suggests adding the corresponding image to each link. Need to talk about this one.

**Comment by andandandandrew** [comment]
We need to discuss this one.

Here are the CTAs we currently have:

Upcoming Shows
About The Band
Latest Music
Latest News

I know we briefly discussed rolling the News & Press into one feed going forward. I'm wondering if the homepage, since we want something else to demonstrate those buttons. I have two ideas:

1: Hover on a button and an image "rises up from behind the button" with a quip of text.
2: Nix the image idea (I don't think pictures of bands on top of the main band picture is a great idea, especially since we've incorporated the instagram feed. Instead I think what we should do is ditch the traditional button look to the four CTA's, and I'll come up with an icon for each and we can center them within that top image piece. Dig? I can mock something up if you need a wireframe to look at.

**Comment by andandandandrew** [comment]
We also need to talk about how that image shrinks because Abie and Brendan are getting cut off on medium sized screens.

**Comment by adamreify** [comment]
Hey hey - 

These CTAs are good still:

Upcoming Shows
About The Band
Latest Music
Latest News

I don't think we should cover up the main image with the CTAs. I still think that it should be a section below the main image but I think the section needs to be larger and more prominent. I think option 2 with icons is cool, and it can still live below the main image. Not sure if that's what you had in mind or if you like the icons over the main image better.  I think a quick mockup is in order before you go developing.

Regarding them getting cut off - I say fuck em!

Just kidding. Maybe there should be a max ratio of height / width. I'll play around with that one.

**Comment by andandandandrew** [comment]
That sounds good to me! Will get on some icons. Do you have any preferences? The Stache is clearly about the band, but the new ones–not sure if you have any sort of bluegrass knowledge you can drop on me to make some genre appropriate ones? Or maybe an alcohol based set? That could be fun. Bottle of bourbon etc...

**Comment by adamreify** [comment]
Yeah I like those ideas.  You could go with whatever makes sense to you but you could try: mustache, bourbon, mandolin, upright bass, treble clef, etc.

Thanks!

**Comment by andandandandrew** [comment]
working on this now

**Comment by andandandandrew** [comment]
popping out for a bit. have an interview and a couple errands. will finish later. we are using an SVG sprite, so we can resize/color/position those suckers in css. I'm switching the scrollstache over to that later too.

**Comment by andandandandrew** [comment]
@adamreify  new icons are in. take a look and if you're cool with them close this issue
